.github/workflows/test-result-comment.yml

```
@@ -13,7 +13,7 @@ jobs:
   comment-test-results:
     name: Publish Test Results
 
-    if: github.event.workflow_run.event == 'pull_request'
+    if: github.event.workflow_run.event == 'pull_request' && !(github.event.workflow_run.conclusion == 'skipped')
```

```suggestion
    if: github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion != 'skipped'
```
maybe for a bit clearer code